prow/scripts/cluster-integration/kyma-gke-integration.sh

```diff
@@ -282,6 +281,8 @@ else
     | kubectl apply -f-
 fi
 
+"${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/generate-ark-config.sh"
```

Comment:
this script is executed by pre-master-kyma-gke-integration job, do we want to always generate ark config and run end to end test?

Reply:
What this script does is to create the secret using the .tlp that is present in Kyma, `installation/resources/ark-secret.yaml.tpl` If this secret is created in another step please let me know where because ark is expecting the secret to contain a `volumeSnapshotLo`cation.provider` a `backupStorageLocation.objectStorage.bucket` and a `credentials.secretContents.cloud` and to be honest I don"t know from where to get this last value

Reply:
Should ark be tested on PR? because you made changes to the script that is running on every PR if you modified resources directory (for example any chart)

Reply:
How can I avoid ark be tested on every PR (What I understand would be happening in the current way). Should I created just a kyma-gke-end-to-end.sh (copy of kyma-gke-integration.sh)  and execute the test from it?

Reply:
yes, you should create separate script

Reply:
@aszecowka I have created the file please take a new look to the PR